src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -408,10 +401,8 @@ private static async Task GetFirstDiagnosticWithFixWithExceptionValidationAsync(
             MockAnalyzerReference? analyzerReference = null,
             TextDocumentKind documentKind = TextDocumentKind.Document)
         {
-            var incrementalAnalyzer = (IIncrementalAnalyzerProvider)diagnosticService;
-
             // register diagnostic engine to solution crawler
-            diagnosticIncrementalAnalyzer = (DiagnosticIncrementalAnalyzer)incrementalAnalyzer.CreateIncrementalAnalyzer(workspace)!;
+            diagnosticIncrementalAnalyzer = diagnosticService.CreateIncrementalAnalyzer(workspace)!;
```

Comment:
the core interfaces went away, so this became strongly typed.  teh tests just test calling through teh actual DiagnosticService and DiagnosticIncrementalAnalyzers now.